etc/nanomq.conf

```diff
@@ -1,94 +1,95 @@
-# NanoMQ Configuration 0.12.2
+# NanoMQ Configuration 0.13.0
 
-##============================================================
-## NanoMQ Broker
-##============================================================
+# #============================================================
+# # NanoMQ Broker
+# #============================================================
 
-url="nmq-tcp://0.0.0.0:1883"
-daemon=false
-num_taskq_thread=0
-max_taskq_thread=0
-parallel=0
-property_size=32
-max_packet_size=1024
-client_max_packet_size=1024
-msq_len=2048
-qos_duration=10
-keepalive_backoff=1250
-allow_anonymous=true
-
-tls {
-     enable=false
-     url="tls+nmq-tcp://0.0.0.0:8883"
-     keyfile="/etc/certs/key.pem"
-     certfile="/etc/certs/cert.pem"
-     cacertfile="/etc/certs/cacert.pem"
-     verify_peer=false
-     fail_if_no_peer_cert=false
+listeners.tcp {
+	enabled = true
+	bind = "0.0.0.0:1883"
+	daemon = false
+	num_taskq_thread = 0
+	max_taskq_thread = 0
+	parallel = 0
+	property_size = 32
+	max_packet_size = 1024
+	client_max_packet_size = 1024
+	msq_len = 2048
+	qos_duration = 10
+	keepalive_backoff = 1250
+	allow_anonymous = true
```

Comment:
numtaskqthread & maxtaskqthread & parallel parameters should not be here. Which could be put to somewhere like sys{}.

Reply:
That's a good idea!